### ISSUE TITLE
netrc for creds

### ISSUE THREAD

**Issue by mcqueenorama** [emicklei/artreyu-nexus#2]
I'm thinking it would be good to support netrc for reading the credentials.  Creds could be omitted from the yaml file, and this nexus plugin would check the netrc for creds for the corresponding url, and use them for the auth. 

This works pretty well:

https://godoc.org/code.google.com/p/go-netrc/netrc

I just want your thoughts on these issues.  I'll do them myself and send some pull requests, once there's a design that makes sense to you.

**Comment by emicklei** [comment]
Yes, let's have support for netrc too and fallback to yaml. I am happy to accept your PR. Thanks

**Comment by mcqueenorama** [comment]
will fork this from walmartlabs repo and send in a pull request
